frontend/src/components/Mobile/styles/ConfirmationModal.styled.ts

```diff
@@ -0,0 +1,70 @@
+import styled from 'styled-components'
+import { Button } from 'antd'
+
+export const OverlayContainer = styled.div<{ overlayBackground: string; isModalOpen: boolean }>`
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  min-height: 100vh;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: ${(props) => (props.overlayBackground ? props.overlayBackground : 'rgba(0, 0, 0, 0.3)')};
+  z-index: 999;
+  ${(props) => !props.isModalOpen && `display: none;`}
+`
+
+export const MainContainer = styled.div<{ isModalOpen: boolean }>`
+  position: fixed;
+  background-color: #fff;
+  width: 90vw;
+  border-radius: 15px;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 15px;
+  box-shadow: 0 2px 5px 1px #888888;
+  z-index: 1000;
+  display: flex;
+  flex-direction: column;
+  ${(props) => !props.isModalOpen && `display: none;`}
+`
+
+export const TitleContainer = styled.div<{ flex?: boolean }>`
+  ${(props) => props.flex && 'display: flex;'}
+`
+
+export const TitleText = styled.text`
+  font-family: Inter;
```

Comment:
I think revamp uses Lato instead? do check figma / w designers!

Reply:
Yep just checked it's Lato, but the Lato font isn't in rhapp yet, probably will remove this line so it defaults to parent font, which should be Lato once we change?